app/connectors/httpParsers/ApiParser.scala

```diff
@@ -33,7 +33,7 @@ import utils.PagerDutyHelper.PagerDutyKeys.{
 }
 import utils.PagerDutyHelper.pagerDutyLog
 
-trait APIParser extends Logging {
+trait APIParserTrait extends Logging {
 
```

Comment:
Having APIParser and ApiParser in the same class was stopping anything from compiling successfully. If you'd rather I did this differently then I can change it